.storybook/blocks/ComponentDetails.jsx

```diff
@@ -264,13 +306,104 @@ function fetchNpmData(packageName, setnpmData, setIsLoading) {
 	}, [cache, setCache, packageName, setnpmData, setIsLoading]);
 }
 
+const fetchLogo = (brand) => {
+	switch(brand) {
+		case "npm":
+			return NpmSVG;
+		case "GitHub":
+			return GitHubSVG;
+		case "Adobe":
+			return AdobeSVG;
+	}
+
+	return;
+}
+
+export const ResourceLinkContent = ({ heading, alt, logo, href }) => {
+	if (!href) return;
+
+	return (
+		<ResourceLink href={href} className="sb-unstyled" title={alt}>
+			<ResourceIconWrapper dangerouslySetInnerHTML={{ __html: fetchLogo(logo) }} />
+			<ResourceTextWrapper>
+				{heading ? <Heading size="xs">{heading}</Heading> : ""}
+				{alt ? <Body size="s">{alt}</Body> : ""}
+			</ResourceTextWrapper>
+		</ResourceLink>
+	);
+};
+
+export const ResourceListDetails = ({ packageName, spectrumData = [], hasDocsPage, title }) => {
+	if (!packageName) return;
+
+	// reformat the component's title so split off "Components/", split off any spaces between words, and capitalize each word. It should resemble the formatting of a rootClass.
+	const reformatTitle = (component) => {
```

Comment:
I added some JSDoc comments. Let me know what you think of them: https://github.com/adobe/spectrum-css/pull/3129/commits/c00c6b6f9959048919bcf364525e0f2bcdec7a3e

Reply:
Looks good; you might want to leave off this part of one of the comments, ", found in packageJson?.spectrum?.guidelines", since any string can be passed in and the function isn't limiting that, and the location could change in the future.